actions/file_actions.jsx

```diff
@@ -17,9 +17,9 @@ export function uploadFile(file, name, channelId, rootId, clientId) {
         return request.
             post(Client4.getFilesRoute()).
             set(Client4.getOptions({method: 'post'}).headers).
-            attach('files', file, name).
             field('channel_id', channelId).
             field('client_ids', clientId).
+            attach('files', file, name).
```

Comment:
Probably worthwhile adding a comment noting that attach needs to be done _after_ `channel_id` to allow for streaming uploads.

Reply:
Yes, forgot to add something. Thanks for reminding me :+1: